qualtran/bloqs/arithmetic/comparison.py

```diff
@@ -1017,3 +1024,135 @@ def _eq_k() -> EqualsAConstant:
 
 
 _EQUALS_K_DOC = BloqDocSpec(bloq_cls=EqualsAConstant, examples=[_eq_k])
+
+
+@frozen
+class CLinearDepthGreaterThan(Bloq):
+    r"""Controlled greater than between two integers.
+
+    Implements |c>|a>|b>|t> => |a>|b>|t ⨁ ((a > b)c)> using $n+2$ Toffoli gates.
+
+    Note: the true cost is $n+1$ but an extra Toffoli comes from OutOfPlaceAdder which operates
+    on $n+1$ qubits rather than $n$. Changing the definition of OutOfPlaceAdder will remove this
```

Comment:
What does changing the definition mean here? Is this a bug, maybe open an issue and link?

Reply:
not a bug just a different meaning ... the outofplaceadder adds to $n$ bit ints into an $n+1$ register ... so the addition is exact ... other bloqs like Add and Subtract do the operation into just $n$ bits (so mod $2^n$)